docs/pages/database-access/auto-user-provisioning/database-object-permissions.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Database object permissions
+description: Understand the concepts behind database object permissions managed by Teleport.
+---
+
+## Overview
+
+Database object permissions is a Teleport feature that enables the configuration of access levels for individual users to specific database objects based on their required level of access.
+
+Database objects are labeled based on import rules, and these labels correspond to specific permissions configured within user roles.
+
+Permissions are granted for the duration of a connection and are automatically revoked upon its closure.
+
+## Database object import rules
+
+A database object import rule in Teleport is a resource that defines the labels to be applied to database objects imported into Teleport. If a specific object does not match any of the rules, it will not be imported.
+
+By default, if no import rules are present, Teleport will automatically create the `import_all_objects` rule on startup:
+
+```yaml
```

Comment:
If we want users to add database object import rules for their own use cases, we should add more comprehensive descriptions of each field. One example is the Access Controls guide for Kubernetes access: https://goteleport.com/docs/kubernetes-access/controls/

If this is an example of the `db_object_import_rule` that we expect most users to get value from, though, I think it's fine to leave this as is.

Reply:
I'd keep it as is for now.